netsim/cli/exec.py

```diff
@@ -0,0 +1,104 @@
+#
+# netlab exec command
+#
+# run a command on one or more lab devices
+#
+import typing
+import os
+import sys
+import argparse
+import subprocess
+from enum import IntEnum
+
+from box import Box
+
+from . import external_commands, set_dry_run
+from . import load_snapshot, _nodeset, parser_add_verbose
+from ..outputs import common as outputs_common
+from ..utils import strings, log
+
+from .connect import quote_list, docker_connect, ssh_connect, LogLevel, get_log_level
+
+
```

Comment:
I would not have more than a single blank line between sections, but that's just my preference.

Reply:
your project, your rules. Ill obey it.